src/common/test-runtime/project.json

```diff
@@ -0,0 +1,43 @@
+{
+  "dependencies": {
```

Comment:
Without this the project `src\Microsoft.DotNet.Build.Tasks.Packaging\test` fails to restore because it is unable to resolve `test-runtime`